### ISSUE TITLE
[addon-knobs] Feature Request - indicate number of knobs in a story

### ISSUE THREAD

**Issue by yairEO** [storybookjs/storybook#11005]
### Describe the solution you'd like

**TLDR;**
When loading a story I would like to know if this story **has any knobs available** *without* clicking each story's _"knobs"_ tab in the story panel.

---

Most of my client's stories don't have *knobs*, but if they do have, it's important to indicate it, while also showing **how many** knobs are there.

Developers at my company (and in general) are **lazy** and will not click the *knobs* tab to see if there's anything there and thus will be unaware of such knobs which are there for manual testing purposes of different UI scenarios. 

Such obvious indicator would be of high value never to miss.

---
### Are you able to assist bring the feature to reality?
no (sorry...)

---
### Screenshot:

![screely-1591086069948](https://user-images.githubusercontent.com/845031/83497515-43b90b80-a4c3-11ea-9dee-0b5b55c7f5aa.png)

**Comment by Attrash-Islam** [comment]
+1 !!

**Comment by yotavm** [comment]
👍

**Comment by yairEO** [comment]
Please implement same logic for `actions` tab as well :)

Sometimes interacting with a story results in _actions addon_ firing actions and it's easily missed unless a number indicator is showing how many actions were triggered.

**Comment by shilman** [comment]
Ooh-la-la!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-beta.24 containing PR #11075 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
